CMSgov/design-system
# [WNMGDS-1713] Date field docs

## Summary

[Jira ticket](https://jira.cms.gov/browse/WNMGDS-1713)

Create documentation based around [Confluence discussion](https://confluence.cms.gov/display/HCDSG/Date+input+feature#tab-Guidelines).

### To test

I tried building a demo site but I'm getting a 403 on the URLs.

To test locally, build both Core and Healthcare docs site (`yarn start` and `yarn start:healthcare`) and view the documentation for all three date field pages. The documentation should match Confluence's guidelines.
